src/api/__tests__/open-service.test.ts

```diff
@@ -13,11 +13,16 @@ describe('api', () => {
     test('myaccount', () => {
       open.mockClear();
 
-      openService(new MtLinkSdk().storedOptions, 'myaccount');
+      const instance = new MtLinkSdk();
+      instance.init('clientId', { samlSubjectId: 'samlSubjectId' });
```

Comment:
Everything is this files seems to modify existing default behaviours tests to a new tests. It should be a separate tests group because both of them are tests different entry point.